experiments/superdarn_common_fields.py

```diff
@@ -84,3 +84,18 @@
 else:
     COMMON_MODE_FREQ_1 = 10400
     COMMON_MODE_FREQ_2 = 13200
+
+# set sounding frequencies
+if opts.site_id == "sas":
+    SOUNDING_FREQS = [9500, 10300, 11000, 11700, 13250, 14200, 15200]
```

Comment:
in case we run this at more than once site at a time, I suggest that we stagger these freqs. For example, leave Saskatoon the way it is but change pgr to be: 10300, 11000, 11700, 13250, 14200, 15200, 9500.
Then shift it again for rkn and again for inv... etc.